src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java

```diff
@@ -80,7 +80,7 @@ public LocalVariableTypeTable(final int name_index, final int length, final Loca
 
     @Override
     public void accept(final Visitor v) {
-        v.visitLocalVariableTypeTable(this);
+        //v.visitLocalVariableTypeTable(this);
```

Comment:
@britter The constructor of MethodGen for existing method remove all local variable information that both LocalVariableTable and LocalVariableTypeTable and then regenerate that. But when regenerate be done LocalVariableTypeTable disappeared and cannot be recovered. 

As like below link LocalVariableTypeTable is optional attribute for type variable or parameterized type.
https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.7.14

Current BCEL implementation cannot regenerate LocalVariableTypeTable so there is no reason to visit LocalVariableTypeTable. I've been faced with JVM crash due to this visiting code and that was fixed after commented out this line. We have been sent this modification to m.dahm@gmx.de at  
2008-05-28 but seems to be not merged repository at that time.

Anyway It's very seldom case. So I cannot suggest ensure test case for reproduce a bug.
It's just my experience before may years ago. The decision is yours.